usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -1492,6 +1491,7 @@ def prepare_orphaned_award_temp_table():
                     list(AWARDS_COLUMNS),
                 ),
                 ("transaction_id", "transaction_id", "id", "id"),
+                strict=False,
```

Comment:
This is the default behavior, right?

Reply:
It is. My formatter always adds this and I usually manually remove it. I missed it this time, but I've removed it in a new commit.